bundle/bundle.go

```diff
@@ -454,6 +532,53 @@ func mktree(path []string, value interface{}) (map[string]interface{}, error) {
 	return dir, nil
 }
 
+// Merge accepts a set of bundles and merges them into a single result bundle. If there are
+// any conflicts during the merge (e.g., with roots) an error is returned. The result bundle
+// will have an empty revision except in the special case where a single bundle is provided.
+func Merge(bundles []*Bundle) (*Bundle, error) {
```

Comment:
Hrmmm. I don't have a strong opinion. I'd probably just leave it as is, unless there's some reason why you think it's bad--I could see the argument that it's simpler to just erase the revision in all cases--then callers _have_ to deal with it...I'm just not sure it simplifies things for the caller...with this approach they can still reset the revision in all cases if they want.

Reply:
> I could see the argument that it's simpler to just erase the revision in all cases--then callers have to deal with it...I'm just not sure it simplifies things for the caller

Well, I guess that simple != least amount of code for the caller. I was thinking that making the caller always have to deal with it seems more likely to result in correct revisions versus them needing to have read the docs and/or understand what it is doing.

Reply:
I've updated the godoc to indicate that in the special case, the bundle is returned unmodified. Thinking about it more, I prefer this because otherwise we need to define properties that determine if a single bundle should be modified at all. For example, should the roots be rewritten? If we add other fields in the future, we could have similar questions to answer. Leaving the bundle unmodified feels like the simplest way to define a "merge" operation on a single bundle. WDYT?

Reply:
Works for me 👍